### PR TITLE
✨ Add scan page

### PR DESCRIPTION
**Pull request by manusmd**
# What I did
I created the first page of the project
The first page includes the following functions: 
- Upload a picture
- Scan picture for text
- Show the recognized text 
- Save document to database

# Design
image

### CODE REVIEW

**Review by JulianSchumacher99 — APPROVED** [review]
🤯

**Review by leonardpaulus — APPROVED** [review]
Lgtm :)

File: src/app/components/Progress/Progress.module.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.progress {
+  display: grid;
+  place-items: center;
+  height: 3rem;
+  background-color: #dfdfdf;
+  border-radius: 1rem;
+}
+
+progress {
```
Comment (lmachens) [inline src/app/components/Progress/Progress.module.css:9]: Be careful using element selectors. This effects all `progress` in your project.

File: src/app/components/Progress/Progress.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  value: number;
+  status: string;
+};
+
+function Progress({ value, status }: ProgressProps): JSX.Element {
+  return (
+    <>
+      <div>
+        <progress className={styles.progressBar} value={value} max="100" />
+      </div>
```
Comment (lmachens) [inline src/app/components/Progress/Progress.tsx:14]: I don't think you need the `div` here

File: src/app/pages/Scan/Scan.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  const { text, progress, recognize, scanActive } = useRecognizeText();
+
+  return (
+    <div className={styles.container}>
+      {text ? <p>{text}</p> : <ImageInput onUpload={setImageURL} />}
+      <div className={styles.buttons}>
+        {scanActive && progress ? (
+          <Progress value={progress.progress * 100} status={progress.status} />
+        ) : (
+          <p></p>
```
Comment (lmachens) [inline src/app/pages/Scan/Scan.tsx:19]: Empty `p`? :)

File: src/app/components/Progress/Progress.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+import Progress from './Progress';
+
+export default {
+  component: Progress,
+  title: 'Component/Progress',
+};
+
+export const ten = () => <Progress status="blabla" value={10} />;
+export const twenty = () => <Progress status="blabla" value={20} />;
+export const thirty = () => <Progress status="blabla" value={30} />;
```
Comment (davidwobido) [inline src/app/components/Progress/Progress.stories.tsx:11]: Interesting status ❤️

**Review by davidwobido — APPROVED** [review]

File: src/app/components/Progress/Progress.module.css (hunk 1 above)
Comment (manusmd) [reply src/app/components/Progress/Progress.module.css:9]: Changed it to the className

File: src/app/components/Progress/Progress.tsx (hunk 2 above)
Comment (manusmd) [reply src/app/components/Progress/Progress.tsx:14]: Deleted the div

File: src/app/pages/Scan/Scan.tsx (hunk 3 above)
Comment (manusmd) [reply src/app/pages/Scan/Scan.tsx:19]: Deleted empty paragraph

**Pull request merged**